src/args.js

```diff
@@ -88,9 +88,9 @@ const args = {
     process.exit(0)
   },
   /**
-   * Calls the cleanup process for Devlab containers and exits
+   * Calls the cleanup process for Binci containers and exits
    */
-  cleanupDL: () => {
+  cleanupBN: () => {
```

Comment:
`cleanBC`?! -- or change other `bc` references to `bn` for consistency as an abbreviation